### ISSUE TITLE
Implement Age 5 cards for Unseen Expansion

### ISSUE THREAD

**Issue by ultimatefiend** [micahstairs/bga-innovation#1078]
This issue will track the implementation of all Age 5 cards for the Unseen expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

- [x] 525 : Popular Science
- [x] 526 : Probability
- [x] 527 : Cabal
- [x] 528 : Witch Trial
- [x] 529 : Buried Treasure
- [x] 530 : Secret History
- [x] 531 : Chartreuse
- [x] 532 : Secret Police
- [x] 533 : Pantheism
- [x] 534 : Pen Name

**Comment by jacksonmead** [comment]
Note that 532 is Museum, not Secret Police.
